Source/WebGPU/WGSL/Metal/MetalFunctionWriter.cpp

```diff
@@ -305,6 +306,10 @@ void FunctionDefinitionWriter::visit(AST::LocationAttribute& location)
     m_stringBuilder.append("[[attribute(", location.location(), ")]]");
 }
 
+void FunctionDefinitionWriter::visit(AST::WorkgroupSizeAttribute&)
+{
```

Comment:
Should we add a comment here as to why this empty?

Reply:
Added